service/src/main/java/org/mskcc/cmo/metadb/service/impl/MessageHandlingServiceImpl.java

```diff
@@ -75,14 +82,78 @@
             new LinkedBlockingQueue<RequestMetadata>();
     private static final BlockingQueue<SampleMetadata> sampleUpdateQueue =
             new LinkedBlockingQueue<SampleMetadata>();
+    private static final BlockingQueue<Map<String, String>> correctCmoPatientIdQueue =
+            new LinkedBlockingQueue<Map<String, String>>();
 
     private static CountDownLatch newRequestHandlerShutdownLatch;
     private static CountDownLatch requestUpdateHandlerShutdownLatch;
     private static CountDownLatch sampleUpdateHandlerShutdownLatch;
+    private static CountDownLatch correctCmoPatientIdShutdownLatch;
     private static Gateway messagingGateway;
 
     private static final Log LOG = LogFactory.getLog(MessageHandlingServiceImpl.class);
 
+    private class CorrectCmoPatientIdReqReplyHandler implements Runnable {
```

Comment:
rename this, since we're not using request-reply anymore

Reply:
good catch